### DISCUSSION TITLE
WineskinCX22 - Status

### DISCUSSION THREAD

**Discussion by Gcenx** [The-Wineskin-Project/WineskinServer#301] [Announcements]
To avoid the usual question I’m just making an announcement.

Yes I’m already working on this, no it’s not going to be done quickly.

~~While I did a quick compile I’m seeing issues that don’t occur with the actual CrossOver-22.0.0 release meaning I’m running into a possible regression in my build environment or I screwed somewhere.~~

The problem wasn’t the environment but some new required CX hacks causing this mess.

**Comment by inzar98** [comment]
Thanks so much for your efforts !! <3

**Comment by sonatique** [comment]
Thanks a lot indeed @Gcenx ! I guess we're quite a few waiting for 22.x engine to be available ;-)
I have to admit I tried to build it myself (based on what you wrote here some time ago : https://forum.winehq.org/viewtopic.php?t=34378) , but I am really stuck with compiling the "32on64". I never tried compiling Wine/WineCX from source before so I can not say wether it's a new problem or not, but I didn't find any available resource that worked for me. The "--with-wine64=" doesn't seem to work...
Anyway, let us know whether we can help!

**Reply by Gcenx** [reply]
> Thanks a lot indeed @Gcenx ! I guess we're quite a few waiting for 22.x engine to be available ;-)

I’m not entirely sure when I’ll have everything fully in-place for CX22 based engines but there at least built.

> I have to admit I tried to build it myself (based on what you wrote here some time ago : https://forum.winehq.org/viewtopic.php?t=34378) , but I am really stuck with compiling the "32on64". I never tried compiling Wine/WineCX from source before so I can not say wether it's a new problem or not, but I didn't find any available resource that worked for me. The "--with-wine64=" doesn't seem to work...

I’m not surprised you’ve having issues building CX sources from CX19 there’s special requirements that are different from upstream wine sources.

If you really want to build it yourself you could make use of [macos-crossover-wine-cloud-builder](https://github.com/GabLeRoux/macos-crossover-wine-cloud-builder), I won’t give support for using the resulting build within Wineskin though.

> Anyway, let us know whether we can help!

Thanks for the offer but these days my time is rather limited due to irl demands (gotta earn that money), outside of possibly setting up say a patreon bit I doubt that would really ever cover working on this full time.

As my Intel MacBook Pro is practically dead (dead battery and dead SSD) meaning  almost all work gets done on the M1 mini that members donated for.

**Reply by sonatique** [reply]
Hey @Gcenx ! Seems like you did it! Great job, thank you very much for your time and efforts!

**Reply by Gcenx** [reply]
@sonatique figured most things are in-place now at least in my winetricks fork, while waiting on another Wineskin wrapper bump would have been idea this will get the ball rolling.
